crates/bevy_ecs/README.md

```diff
@@ -68,18 +87,20 @@ fn print_position(query: Query<(Entity, &Position)>) {
 Apps often require unique resources, such as asset collections, renderers, audio servers, time, etc. Bevy ECS makes this pattern a first class citizen. `Resource` is a special kind of component that does not belong to any entity. Instead, it is identified uniquely by its type:
 
 ```rust
+use bevy_ecs::prelude::*;
+
 #[derive(Default)]
-struct Time {
-    seconds: f32,
```

Comment:
I object to using a tuple struct here. The named field is way better at documenting intent. Tuple structs rarely have a place in public apis imo.

Reply:
Sounds good to me 👍